### PR TITLE
#753. rest api integration test check response headers, check error m…

### PR DESCRIPTION
**Pull request by madryk**
…essages

### CODE REVIEW

File: saos-restful-web-api/src/test/java/pl/edu/icm/saos/api/ApiResponseAssertUtils.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    public static void assertError(ResultActions result, HttpStatus httpStatus, String reason,
+            String propertyName, Matcher<String> messageMatcher) throws Exception {
+        
+        result.andExpect(status().is(httpStatus.value()))
+                .andExpect(header().string("content-type", containsString("application/json")))
+                .andExpect(jsonPath("$.error.httpStatus").value(String.valueOf(httpStatus.value())))
+                .andExpect(jsonPath("$.error.reason").value(reason))
+                .andExpect(jsonPath("$.error.propertyName").value(propertyName))
+                .andExpect(jsonPath("$.error.message").value(messageMatcher));
+    }
```
Comment (pawelczak) [inline saos-restful-web-api/src/test/java/pl/edu/icm/saos/api/ApiResponseAssertUtils.java]: I think it's better to move this method to top part of class, just for better reading.

File: saos-restful-web-api/src/test/java/pl/edu/icm/saos/api/ApiResponseAssertUtils.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+import org.springframework.test.web.servlet.ResultActions;
+
+import pl.edu.icm.saos.api.services.exceptions.status.ErrorReason;
+
+/**
+ * @author madryk
+ */
+public class ApiResponseAssertUtils {
+
+
```
Comment (pawelczak) [inline saos-restful-web-api/src/test/java/pl/edu/icm/saos/api/ApiResponseAssertUtils.java]: Missing code section logic

File: saos-restful-web-api/src/test/java/pl/edu/icm/saos/api/dump/supreme/court/chamber/DumpSupremeCourtChambersControllerTest.java [hunk 3]
```diff
@@ -16,14 +23,21 @@
...
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 import org.springframework.test.web.servlet.MockMvc;
 import org.springframework.test.web.servlet.ResultActions;
 
+
+
+
+
+//import pl.edu.icm.saos.api.ApiConstants;
```
Comment (pawelczak) [inline saos-restful-web-api/src/test/java/pl/edu/icm/saos/api/dump/supreme/court/chamber/DumpSupremeCourtChambersControllerTest.java]: Commented line.

Comment (madryk) [inline saos-restful-web-api/src/test/java/pl/edu/icm/saos/api/dump/supreme/court/chamber/DumpSupremeCourtChambersControllerTest.java]: :+1:  I will remove it

File: saos-restful-web-api/src/test/java/pl/edu/icm/saos/api/ApiResponseAssertUtils.java (hunk 2 above)
Comment (madryk) [inline saos-restful-web-api/src/test/java/pl/edu/icm/saos/api/ApiResponseAssertUtils.java]: ok

File: saos-restful-web-api/src/test/java/pl/edu/icm/saos/api/ApiResponseAssertUtils.java (hunk 1 above)
Comment (madryk) [inline saos-restful-web-api/src/test/java/pl/edu/icm/saos/api/ApiResponseAssertUtils.java]: maybe you're right :+1:

**Pull request closed without merging**